deployments/multus-daemonset-thick.yml

```diff
@@ -190,7 +190,7 @@ spec:
               readOnly: true
             - name: hostroot
               mountPath: /hostroot
-              mountPropagation: HostToContainer
+              mountPropagation: Bidirectional
```

Comment:
don't we actually consider these yamls to be just examples ? 

Why can't the downstreams that require it to be bi-directional use what they need ? 

I'm trying to wrap my head around the fact this is not the recommended configuration, required only for specific use cases, but still, we're advocating / instructing everyone to use it.

Reply:
These are indeed examples, but, I think this more accurately represents a setup that's similar to what happens in a thin plugin mode on the host -- you'd see mounts in both directions. So, I think it's a valid setup and probably better for more cases at least.

Reply:
I disagree, but don't have more arguments other than this is not the recommended configuration, and is required only for specific use cases.

Do we even know if this will work on openshift ?